src/app/components/Grid/index.jsx

```diff
@@ -21,4 +23,138 @@ export const GelPageGrid = styled(Grid)`
   ${gelMaxWidths}
 `;
 
+/* This is intented to be used on pages with single columns
+ * i.e. ArticlePage, PhotoGalleryPage and MediaAssetPage
+ */
+export const SingleColumnWrapper = styled.div`
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+    margin: 0 auto;
+    max-width: 46.4rem;
+  }
+`;
```

Comment:
Is there a reason why the margin and max-width are applied on a separate div here instead of being defined here as a `css` style and included in the `GridWrapperMedium` below?
Is `GridWrapperMedium` going to be used at any level other than the page level?